### PR TITLE
29 fix footer

### PR DESCRIPTION
**Pull request by hodorogandrei**
Added the "Footer ads" widget area.
Puteti acum adauga sigla de la Agora si alti parteneri din admin

### CODE REVIEW

File: wp-content/themes/hueman_child/style.css [hunk 1]
```diff
@@ -38,4 +38,8 @@ License URI: http://www.gnu.org/licenses/gpl-3.0.html
 
 #nav-header.nav-container {
   background-color: #666;
+}
+
+.entry.excerpt { 
+  color: #666;
```
Comment (vtemian) [inline wp-content/themes/hueman_child/style.css:44]: Does this change effect footer issue?
In a PL, changes should be related to the same issue

Comment (hodorogandrei) [inline wp-content/themes/hueman_child/style.css:44]: Nu, nu are legatura...asta face mai accentuata culoarea fontului articolelor (am discutat cu robert.dolca azi dimineata)

Comment (robertdxyz) [inline wp-content/themes/hueman_child/style.css:44]: Please remove the last commit and make another PR
